modules/edge-machine-pools-aws-local-zones.adoc

```diff
@@ -4,5 +4,20 @@
 
 Edge worker nodes are tainted worker nodes that run in AWS Local Zones locations.
 
+There are some important information when deploying a cluster using Local Zones:
+
 * Amazon EC2 instances in the Local Zones are more expensive than Amazon EC2 instances in the Availability Zones
-* Latency between applications and end users are lower in Local Zones, and it may vary by location. There is a latency impact for some workloads if, for example, routers are mixed between Local Zones and Availability Zones.
\ No newline at end of file
+* Latency between applications and end users are lower in Local Zones, and it may vary by location. There is a latency impact for some workloads if, for example, routers are mixed between Local Zones and Availability Zones.
+* The cluster-network Maximum Transmission Unit (MTU) is adjusted automatically to the lower restricted by AWS when Local Zone subnets are detected on the `install-config.yaml`, according to the network plugin.
```

Comment:
This line is aligned with 4.12 documentation, adapting to 4.13  - which we don't need to change it manually:

https://docs.openshift.com/container-platform/4.12/installing/installing_aws/installing-aws-localzone.html#installation-localzone-generate-k8s-manifest_installing-aws-localzone

Reply:
```suggestion
* The cluster-network Maximum Transmission Unit (MTU) is adjusted automatically to the lower end restricted by AWS when Local Zone subnets are detected on the `install-config.yaml` file, according to the network plugin.
```

Having trouble parsing this. Is it:

> When Local Zone subnets are present in the `install-config.yaml` file, the network plugin adjusts the cluster-network's Maximum Transmission Unit (MTU) to the lower end restricted by AWS. This adjustment happens automatically.

Reply:
- The *installer* (program) adjusts the *MTU* (attribute) based on the network plugin used on the cluster (component)

Reply:
```suggestion
* When Local Zone subnets are present in the `install-config.yaml` file, the installer adjusts the MTU based on the network plugin that is used on the cluster.
```

Reply:
Yes, it's the most critical and important to people know that the MTU will be changed from 9000 to lower (1200, 1250, etc - depends on the network plugin).
I will commit that way, let me know if you can adjust based on that information.

Reply:
Shouldn't be a problem.

Reply:
sgtm